Source/WebKit/UIProcess/_WKTouchEventGenerator.mm

```diff
@@ -121,8 +122,8 @@ - (instancetype)init
     if (!self)
         return nil;
 
-    for (NSUInteger i = 0; i < HIDMaxTouchCount; ++i)
-        _activePoints[i].identifier = fingerIdentifiers[i];
+    for (auto [i, fingerIdentifier] : IndexedRange(fingerIdentifiers))
```

Comment:
Still hoping for something zippier in the future, but I suppose what’s really important to avoid buffer overruns is the runtime index checking.